cloud/api-gateway/src/main/resources/application-local.yaml

```
@@ -1,4 +1,28 @@
 server:
   port: 8080
   error:
-    include-message: always
\ No newline at end of file
+    include-message: always
+spring:
+  cloud:
+    gateway:
+      routes:
+        - id: user-manager
+          uri: http://localhost:8081
+          predicates:
+            - Path=/api/v1/customers/**,/api/v1/merchants/**
+        - id: payment-manager
+          uri: http://localhost:8082
+          predicates:
+            - Path=/api/v1/wallets/**,/api/v1/payments/**
+        - id: notification-manager
+          uri: http://localhost:8083
+          predicates:
+            - Path=/api/v1/notifications/**
+        - id: qr-generator
+          uri: http://localhost:8085
+          predicates:
+            - Path=/api/v1/qrcode/**
+        - id: auth-manager
+          uri: http://localhost:8086
+          predicates:
+            - Path=/api/v1/users/**,/api/v1/auth/**
```

```suggestion
            - Path=/api/v1/auth/users/**,/api/v1/auth/**
```
I don't like that for auth manager the path is `/api/v1/users/` .  Looks like it is for user-manager.